GL_Framework/tests/api_test/test_login.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+from libs.api_client import ApiClient
+from configuration.config import ConfigModel, get_config_variable_by_name
+from assertpy import assert_that, soft_assertions
+from helpers.fake import fake
+
+@pytest.mark.smoke
+def test_login():
+    resp = ApiClient.loggin(
+        get_config_variable_by_name(ConfigModel.user),
+        get_config_variable_by_name(ConfigModel.password))
+    assert_that(resp.status_code).is_equal_to(200)
+
+@pytest.mark.smoke
+def test_invalide_login():
+    name = fake.name()
+    password = fake.password()
+    resp = ApiClient.loggin(
+        name,
+        password)
+    with soft_assertions():
+        assert_that(resp.status_code).is_equal_to(401)
```

Comment:
Expected status codes should not be hardcoded, keep them in constants in configuration / data files